### ISSUE TITLE
[Player] Fix soft keyboard not showing after 2nd time editing a player

### ISSUE THREAD

**Issue by Chiniga** [Chiniga/pa-queue-ph#10]
Soft keyboard only shows automatically on 1st time editing a player. It should be every time.

**Comment by Chiniga** [comment]
Removed functionality since user may only want to change player level
